### PR TITLE
3.0 preview dev

### PR DESCRIPTION
**Pull request by notwaldorf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,84 @@
+[![Published on NPM](https://img.shields.io/npm/v/@polymer/iron-validator-behavior.svg)](https://www.npmjs.com/package/@polymer/iron-validator-behavior)
+[![Build status](https://travis-ci.org/PolymerElements/iron-validator-behavior.svg?branch=master)](https://travis-ci.org/PolymerElements/iron-validator-behavior)
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://webcomponents.org/element/@polymer/iron-validator-behavior)
 
-<!---
+## &lt;iron-validator-behavior&gt;
```
Comment (bicknellr) [inline README.md]: `&lt;iron-validator-behavior&gt;` -> `` `IronValidatorBehavior` ``

File: iron-validator-behavior.js [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+part of the polymer project is also subject to an additional IP rights grant
+found at http://polymer.github.io/PATENTS.txt
 */
 import '@polymer/polymer/polymer-legacy.js';
 
-import { IronMeta } from '@polymer/iron-meta/iron-meta.js';
+import {IronMeta} from '@polymer/iron-meta/iron-meta.js';
 
 /**
  * Use `Polymer.IronValidatorBehavior` to implement a custom input/form
```
Comment (bicknellr) [inline iron-validator-behavior.js]: `Polymer.IronValidatorBehavior` -> `IronValidatorBehavior`; there's another just below here.

File: README.md [hunk 3]
```diff
@@ -1,27 +1,84 @@
...
+```js
+import {PolymerElement, html} from '@polymer/polymer';
+import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
+import {IronValidatorBehavior} from '@polymer/iron-validator-behavior/iron-validator-behavior.js';
 
-[![Build status](https://travis-ci.org/PolymerElements/iron-validator-behavior.svg?branch=master)](https://travis-ci.org/PolymerElements/iron-validator-behavior)
+class SampleValidator mixinBehaviors([IronValidatorBehavior], PolymerElement){
+  static get template() {
+    return html``;
+  }
```
Comment (bicknellr) [inline README.md]: Is the `template` getter necessary if this element just returns an empty template?

File: README.md [hunk 4]
```diff
@@ -1,27 +1,84 @@
...
 
--->
+### In a Polymer 3 element that is a validator
+```js
+import {PolymerElement, html} from '@polymer/polymer';
+import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
+import {IronValidatorBehavior} from '@polymer/iron-validator-behavior/iron-validator-behavior.js';
 
-[![Build status](https://travis-ci.org/PolymerElements/iron-validator-behavior.svg?branch=master)](https://travis-ci.org/PolymerElements/iron-validator-behavior)
+class SampleValidator mixinBehaviors([IronValidatorBehavior], PolymerElement){
```
Comment (bicknellr) [inline README.md]: Missing `extends`.

File: README.md [hunk 5]
```diff
@@ -1,27 +1,84 @@
...
+    <script type="module">
+      import '../sample-validator.js';
+    </script>
+  </head>
+  <body>
+    <sample-validator id="aValidator"></sample-validator>
+    <input id="input">
+    <script>
+      input.addEventListener('input', function(event) {
+        var valid = simpleCats.validate(input.value);
```
Comment (bicknellr) [inline README.md]: `simpleCats` -> `aValidator`, I think?

**Review by bicknellr — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (notwaldorf) [reply README.md]: i don't know! ill delete it

**Review by bicknellr — APPROVED** [review]

**Pull request merged**